tests/optimizer/autograd/common.py

```diff
@@ -0,0 +1,30 @@
+import torch
```

Comment:
Not sure if this was added later but this file is missing license header in this PR.